bayes_search.py

```diff
@@ -273,43 +259,47 @@ def next_sample(
     min_unnorm_y = np.min(filtered_y)
     # hack for dealing with predicted std of 0
     epsilon = 0.00000001
+
     if opt_func == "probability_of_improvement":
-        # might remove the norm_improvement at some point
-        # find best chance of an improvement by "at least norm improvement"
-        # so if norm_improvement is zero, we are looking for best chance of any
-        # improvment over the best result observerd so far.
-        # norm_improvement = improvement / y_stddev
         min_norm_y = (min_unnorm_y - y_mean) / y_stddev - improvement
-        std_dev_distance = (y_pred - min_norm_y) / (y_pred_std + epsilon)
-        prob_of_improve = sigmoid(-std_dev_distance)
-        best_test_X_index = np.argmax(prob_of_improve)
-    elif opt_func == "expected_improvement":
+    else:
         min_norm_y = (min_unnorm_y - y_mean) / y_stddev
-        Z = -(y_pred - min_norm_y) / (y_pred_std + epsilon)
-        prob_of_improve = scipy_stats.norm.cdf(Z)
-        e_i = -(y_pred - min_norm_y) * scipy_stats.norm.cdf(
-            Z
-        ) + y_pred_std * scipy_stats.norm.pdf(Z)
-        best_test_X_index = np.argmax(e_i)
+
+    Z = -(y_pred - min_norm_y) / (y_pred_std + epsilon)
+    prob_of_improve: np.ndarray = scipy_stats.norm.cdf(Z)
+    e_i = -(y_pred - min_norm_y) * scipy_stats.norm.cdf(
+        Z
+    ) + y_pred_std * scipy_stats.norm.pdf(Z)
+
+    if opt_func == "probability_of_improvement":
```

Comment:
can you add a "probability_of_improvement" unittest, or comment our or note that this isnt live code

Reply:
done